ftw/dashboard/portlets/favourites/browser/favourite.py

```diff
@@ -1,14 +1,23 @@
-import json
-import time
-
-from zope.component import getMultiAdapter
+from ftw.dashboard.portlets.favourites import favouriteMessageFactory as _
+from ftw.dashboard.portlets.favourites.interfaces import IFavouritesHandler
+from plone.i18n.normalizer.interfaces import IIDNormalizer
 from Products.CMFCore.utils import getToolByName
 from Products.Five.browser import BrowserView
+from zope.component import getMultiAdapter
 from zope.component import getUtility
-from plone.i18n.normalizer.interfaces import IIDNormalizer
+import json
+import time
 
-from ftw.dashboard.portlets.favourites import favouriteMessageFactory as _
-from ftw.dashboard.portlets.favourites.interfaces import IFavouritesHandler
+
+# Try to get the plone.protect's createToken method, because it's only
+# available since version 2.0.2, otherwise we just use a mocked method.
+try:
+    from plone.protect import createToken
+    PLONE_PROTECT_AVAILABLE = True
+except ImportError:
+    PLONE_PROTECT_AVAILABLE = False
+    def createToken():
```

Comment:
you don't need this, you only call the method if `PLONE_PROTECT_AVAILABLE` is `True`